pkg/solana/relay.go

```diff
@@ -65,31 +71,25 @@ type OCR2Spec struct {
 	NodeEndpointRPC string
 	NodeEndpointWS  string
 
-	// on-chain program + 2x state accounts (state + transmissions)
-	ProgramID       solana.PublicKey
-	StateID         solana.PublicKey
-	TransmissionsID solana.PublicKey
+	// on-chain program + 2x state accounts (state + transmissions) + validator program
+	ProgramID          solana.PublicKey
+	StateID            solana.PublicKey
+	ValidatorProgramID solana.PublicKey
+	TransmissionsID    solana.PublicKey
```

Comment:
Isn't this available from the `Transmitter` interface as `t.PublicKey()`?

Reply:
@krebernisak not sure i understand your question...
`TransmissionsID` refers to the account where the transmissions state is stored...`Transmitter` is the account sending the report tx.

Reply:
Ah, yes...

Reply:
Was thinking about the `Transmitter` interface name, and then this confused me. Thx!